system-x/common/src/main/java/software/tnb/common/deployment/WithOperatorHub.java

```diff
@@ -50,12 +55,21 @@ default boolean clusterWide() {
     }
 
     default void createSubscription() {
+        final AtomicReference<SubscriptionConfig> config = new AtomicReference<>();
```

Comment:
why is the atomic reference needed here?

Reply:
now I refactored to avoid to declare it, thanks